base_layer/wallet/Cargo.toml

```diff
@@ -37,7 +37,7 @@ chrono = { version = "0.4.19", default-features = false, features = ["serde"] }
 clear_on_drop = "=0.2.4"
 crossbeam-channel = "0.5.4"
 derivative = "2.2.0"
-diesel = { version = "1.4.8", features = ["sqlite", "serde_json", "chrono"] }
+diesel = { version = "1.4.8", features = ["sqlite", "serde_json", "chrono", "64-column-tables"] }
```

Comment:
This usually increases compile time by a large amount. Can we drop a column maybe to fit into 32?

Reply:
The CI didn't seem to take too long, so maybe not so bad

Reply:
Yeah we can drop  parent_pk and/or unique_id later and remove this feature.